ViewModel/BrowserViewModel.swift

```diff
@@ -158,15 +159,25 @@ final class BrowserViewModel: NSObject, ObservableObject,
         }
     }
 
+    func pdfData() async -> Data? {
+        if metaData?.isTextType == true {
```

Comment:
This mandates a comment. I believe this triggers PDF generation of the displayed page if the entry's type is text (HTML, etc). Right? Otherwise it returns the data from libzim so this returns binary anyway I guess